<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjeto</class>
 <widget class="QDialog" name="AddObjeto">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adcionar novo objeto</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="base_verticalLayout">
     <item>
      <layout class="QFormLayout" name="nome_formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nomeLabel">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameField"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="tabs_horizontalLayout">
       <item>
        <widget class="QTabWidget" name="objetosTab">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pontoTab">
          <attribute name="title">
           <string>Ponto</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="pontoBase_verticalLayout">
             <item>
              <widget class="QGroupBox" name="pontoCoordenadas_groupBox">
               <property name="title">
                <string>Coordenadas</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="pCoord_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="pX_label">
                    <property name="text">
                     <string>X</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="pointX_field"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="pY_label">
                    <property name="text">
                     <string>Y</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="pointY_field"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="pZ_label">
                    <property name="text">
                     <string>Z</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="pointZ_field"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="wireTab">
          <attribute name="title">
           <string>Reta</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="retaBase_verticalLayout">
             <item>
              <widget class="QGroupBox" name="coordenadaIni_groupBox">
               <property name="title">
                <string>Coordenadas do ponto inicial</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="retaCoordIni_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="retaXIni_label">
                    <property name="text">
                     <string>X</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireXIni_field"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="retaYIni_label">
                    <property name="text">
                     <string>Y</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireYIni_field"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="retaZIni_label">
                    <property name="text">
                     <string>Z</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireZIni_field"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="coordenadaFinal_groupBox">
               <property name="title">
                <string>Coordenadas do ponto final</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="retaCoordFinal_horizontalLayout">
                  <item>
                   <widget class="QLabel" name="retaXFinal_label">
                    <property name="text">
                     <string>X</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireXFinal_field"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="retaYFinal_label">
                    <property name="text">
                     <string>Y</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireYFinal_field"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="retaZFinal_label">
                    <property name="text">
                     <string>Z</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireZFinal_field"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="wireframeTab">
          <attribute name="title">
           <string>Wireframe</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Coordenadas do ponto inicial</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="wireXini_label">
                    <property name="text">
                     <string>X</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireframeXIni"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="wireYini_label">
                    <property name="text">
                     <string>Y</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireframeYIni"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="wireZini_label">
                    <property name="text">
                     <string>Z</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireframeZIni"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Coordenadas do ponto final</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="wireXFinal_label">
                    <property name="text">
                     <string>X</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireframeXFinal"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="wireYFinal_label">
                    <property name="text">
                     <string>Y</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireframeYFinal"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="wireZFinal_label">
                    <property name="text">
                     <string>Z</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="wireframeZFinal"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="horizontalFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>29</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="numArestas_label">
                  <property name="text">
                   <string>Arestas:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QListWidget" name="arestasAdcionadas"/>
                </item>
                <item>
                 <widget class="QPushButton" name="addAresta">
                  <property name="text">
                   <string>Adiciona aresta</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="wireFramePreenchido">
                  <property name="text">
                   <string>Preenchido</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="curvaTab">
          <attribute name="title">
           <string>Curva Hermite</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_20">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>P1X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveP1X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_21">
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>P1Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveP1Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_22">
                 <item>
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>P4X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveP4X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_23">
                 <item>
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>P4Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveP4Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_18">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>R1X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveR1X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_19">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>R1Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveR1Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>R4X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveR4X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>R4Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveR4Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Precisão da curva: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="precisaoCurva">
               <property name="text">
                <string>100</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="curvasContinuasTab">
          <attribute name="title">
           <string>Curva Continuas Hermite</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_24">
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>P1X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousP1X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_25">
                 <item>
                  <widget class="QLabel" name="label_16">
                   <property name="text">
                    <string>P1Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousP1Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_26">
                 <item>
                  <widget class="QLabel" name="label_17">
                   <property name="text">
                    <string>P4X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousP4X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_27">
                 <item>
                  <widget class="QLabel" name="label_18">
                   <property name="text">
                    <string>P4Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousP4Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_28">
                 <item>
                  <widget class="QLabel" name="label_19">
                   <property name="text">
                    <string>R1X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousR1X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_29">
                 <item>
                  <widget class="QLabel" name="label_20">
                   <property name="text">
                    <string>R1Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousR1Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_16">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>R4X</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousR4X"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_17">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>R4Y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="curveContinuousR4Y"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Número Curvas: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="numeroCurvas">
               <property name="text">
                <string>0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="adcionarCurva">
               <property name="text">
                <string>Adcionar Curva</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_30">
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Precisão da curva: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="precisaoCurvaContinua">
               <property name="text">
                <string>100</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="bsplineTab">
          <attribute name="title">
           <string>BSpline Curve</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_42">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_43">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_17">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_45">
                   <item>
                    <widget class="QLabel" name="bsX_label">
                     <property name="text">
                      <string>X:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="bsplineX"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="bsY_label">
                     <property name="text">
                      <string>Y:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="bsplineY"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_46">
                   <item>
                    <widget class="QPushButton" name="addPointBSpline">
                     <property name="text">
                      <string>Adicionar ponto</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QListWidget" name="bsplineCoordinates"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="colorSelect_pushButton">
       <property name="text">
        <string>Selecione a cor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="confirmaObj_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmaObj_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddObjeto</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmaObj_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddObjeto</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
